src/WebJobs.Extensions/Extensions/Timers/Listener/TimerListener.cs

```diff
@@ -84,7 +84,7 @@ internal System.Timers.Timer Timer
                 // check to see if we've missed an occurrence since we last started.
                 // If we have, invoke it immediately.
                 ScheduleStatus = await ScheduleMonitor.GetStatusAsync(_timerName);
-                _trace.Verbose($"Function '{_timerName}' initial status: Last='{ScheduleStatus?.Last.ToString("o")}', Next='{ScheduleStatus?.Next.ToString("o")}', LastUpdated='{ScheduleStatus?.LastUpdated.ToString("o")}'");
+                _logger.LogDebug($"Function '{_timerName}' initial status: Last='{ScheduleStatus?.Last.ToString("o")}', Next='{ScheduleStatus?.Next.ToString("o")}', LastUpdated='{ScheduleStatus?.LastUpdated.ToString("o")}'");
```

Comment:
Doesn't LogTrace map to Verbose?

Reply:
Verbose could map to either Trace or Debug as LogLevel has more possibilities than TraceLevel does. I believe I've chosen Debug everywhere. I was reserving Trace for even more detailed logging which we may want to add in the future.